autodoc/data/admin-api.lua

```diff
@@ -611,6 +611,14 @@ return {
             strip the matching prefix from the upstream request URL.
           ]]
         },
+        path_handling = {
+          description = [[
+            Controls how the Service Path, Route Path and requested path are combined when sending a request to the
+            upstream. While these often will be just concatenated, there are some edge cases where they will not; for
+            instance, Kong will remove slashes to avoid sending "double slashes" on the request.
+            `"v0"` is the behavior used in Kong 0.x. `"v1"` is the behavior used in Kong 1.x.
```

Comment:
The most important thing to document is how the concatenation happens:
`v0` concatenates `service.path` to possible stripped path by ensuring there is a single `/` between them
`v1` concatenates `service.path` to possible stripped path without `/` separator.

`v0` treats the `service.path` as a path with complete `segments`
`v1` treats the `service.path` as a `prefix`

I think the current description as of now gives little value to reader, and he needs to perhaps go to code and try to figure out. Yes, we could document the bugs in them too, like how they endup removing chars from `service.path` in some cases.

Reply:
I have done my best to document this as requested on this comment. Leaving this open in case it is not satisfactory.